packages/babel-plugin-transform-modules-systemjs/test/fixtures/preset-env/requeue-helpers/input.js

```diff
@@ -0,0 +1,39 @@
+// Ref: https://github.com/babel/babel/issues/16219
+
+// delete global.Symbol doesn't work with jest in node 6
+Object.defineProperty(global, "Symbol", {
+  configurable: true,
+  writable: true,
+  value: undefined
+});
```

Comment:
To be honest, I don't understand how jest affects our isolation environment in node 6.

Reply:
Jest does bad things to the global object to implement isolation-but-not-too-much, this workaround is good enough.